packages/e2e-tests/specs/editor/various/change-detection.test.js

```diff
@@ -1,417 +0,0 @@
-
-		await publishPost();
-
-		// Close publish panel.
-		await Promise.all( [
-			page.waitForFunction(
-				() => ! document.querySelector( '.editor-post-publish-panel' )
-			),
-			page.click( '.editor-post-publish-panel__header button' ),
-		] );
-
-		// Should be dirty after autosave change of published post.
-		await canvas().type( '.editor-post-title__input', '!' );
-
-		await Promise.all( [
-			page.waitForSelector(
-				'.editor-post-publish-button[aria-disabled="true"]'
-			),
-			page.waitForSelector(
-				'.editor-post-publish-button[aria-disabled="false"]'
-			),
-			page.evaluate( () =>
-				window.wp.data.dispatch( 'core/editor' ).autosave()
-			),
-		] );
-
-		await assertIsDirty( true );
-	} );
-
-	it( 'Should not prompt to confirm unsaved changes for new post', async () => {
-		await assertIsDirty( false );
-	} );
-
-	it( 'Should prompt to confirm unsaved changes for new post with initial edits', async () => {
-		await createNewPost( {
-			title: 'My New Post',
-			content: 'My content',
-			excerpt: 'My excerpt',
-		} );
-
-		await assertIsDirty( true );
-	} );
-
-	it( 'Should prompt if property changed without save', async () => {
-		await canvas().type( '.editor-post-title__input', 'Hello World' );
-
-		await assertIsDirty( true );
-	} );
-
-	it( 'Should prompt if content added without save', async () => {
-		await clickBlockAppender();
-		await page.keyboard.type( 'Paragraph' );
-
-		await assertIsDirty( true );
-	} );
-
-	it( 'Should not prompt if changes saved', async () => {
-		await canvas().type( '.editor-post-title__input', 'Hello World' );
-
-		await saveDraft();
-
-		await assertIsDirty( false );
-	} );
-
-	it( 'Should not prompt if changes saved right after typing', async () => {
-		await clickBlockAppender();
-		await page.keyboard.type( 'Hello World' );
-
-		await saveDraft();
-
-		await assertIsDirty( false );
-	} );
-
-	it( 'Should not save if all changes saved', async () => {
-		await canvas().type( '.editor-post-title__input', 'Hello World' );
-
-		await saveDraft();
-
-		await interceptSave();
-
-		// Keyboard shortcut Ctrl+S save.
-		await pressKeyWithModifier( 'primary', 'S' );
-
-		expect( hadInterceptedSave ).toBe( false );
-	} );
-
-	it( 'Should prompt if save failed', async () => {
-		await canvas().type( '.editor-post-title__input', 'Hello World' );
-
-		await page.setOfflineMode( true );
-
-		await Promise.all( [
-			// Keyboard shortcut Ctrl+S save.
-			pressKeyWithModifier( 'primary', 'S' ),
-
-			// Ensure save update fails and presents button.
-			page.waitForXPath(
-				'//*[contains(@class, "components-notice") and contains(@class, "is-error")]/*[text()="Updating failed. You are probably offline."]'
-			),
-			page.waitForSelector( '.editor-post-save-draft' ),
-		] );
-
-		// Need to disable offline to allow reload.
-		await page.setOfflineMode( false );
-
-		await assertIsDirty( true );
-
-		expect( console ).toHaveErroredWith(
```

Comment:
This is left out in the migrated test since this is browser-dependent. I don't think it provides much value to the test.